content/doc/pipeline/jenkinsfile.adoc

```diff
@@ -50,9 +50,9 @@ A pull reqeust to a repository included in or monitored by an Organization Folde
 
 == Using Organization Folders
 
-Organization folders enable Jenkins to automatically detect and include any new repositories within them as resources. 
+Organization folders enable Jenkins to automatically detect and include any new repositories within them as resources.
 
-When you create a new repository (as might be the case for a new project), that repository has a Jenkinsfile. If you also configure one or more organization folders, Jenkins automatically detects any repository in an organization folder, scans the contents of that repository at either default or configurable intervals, and creates a Multi-branch Pipeline project for what it finds in the scan. Organzation folders and multi-branch pipelines 
+When you create a new repository (as might be the case for a new project), that repository has a Jenkinsfile. If you also configure one or more organization folders, Jenkins automatically detects any repository in an organization folder, scans the contents of that repository at either default or configurable intervals, and creates a Multi-branch Pipeline project for what it finds in the scan. Organzation folders and multi-branch pipelines
```

Comment:
The line "Organzation folders and multi-branch pipelines" could be a new heading, but best to delete it because the paragraph that it was at the end of by mistake is already part of a section with the heading "Using Organization Folders"